new-website/src/components/location/location-table.jsx

```diff
@@ -74,7 +116,7 @@ export const LocationTable = ({
 
     const onRowUpdate = updateRow(onError)
     const onRowDelete = deleteRow(onError)
-    const onRowAdd = addNewRow(state.userAllowedPartnerIds, onError)
+    const onRowAdd = addNewRow(state.userAllowedPartnerIds, state.partnerConfigs, onError)
```

Comment:
If we send the user to a new page .. we can either hand over all the infos or fetch them there from the useSession() hook